packages/perseus/src/widgets/image.tsx

```diff
@@ -243,23 +239,23 @@ class ImageWidget extends React.Component<Props> {
 
         if (this.props.caption) {
             caption = (
-                <div className="perseus-image-caption">
+                <figcaption className="perseus-image-caption">
```

Comment:
Not related to your changes, but kind of a bummer we're repeating some of this markup. Seems like it should be in one spot so when we're changing things (like now) we only need to change one thing.